main.go

```diff
@@ -27,6 +28,16 @@ func main() {
 	nowFlag := flag.Bool("now", false, "Flag to indicate if the integrator should run immediately (env: INTEGRATOR_NOW, default: false).  If used without cron, integrator will run once and then exit.  If now is not set, \"cron\" must be supplied.")
 	flag.Parse()
 
+	err := os.Mkdir("/etc/integratorlogs", 0755)
+	if err != nil && !os.IsExist(err){
+		fmt.Println("Error creating log directory:" + err.Error())
+	}
+	lf, err := os.Create("/etc/integratorlogs/integrator.log")
```

Comment:
`os.Create` will create a new file or _truncate_ and existing file.  That means that if the service is restarted, all the existing logs in the file will be deleted.  I don't think that's what we want.  We want to create it if it doesn't exist, otherwise we want to append it.